src/components/index.ts

```diff
@@ -90,3 +90,6 @@ export * from './section';
 
 export { default as TimeInput } from './time-input';
 export * from './time-input';
+
+export { default as ProgressBar } from './progress-bar';
+export * from './progress-bar';
```

Comment:
It looks like most of this file is sorted alphabetically so we can try to add the entry after `PrimaryButton`: https://github.com/box/box-ui-elements/blob/master/src/components/index.ts#L1-L74

Reply:
"most of it" :) let's sort them all then

Reply:
fixed